packages/dd-trace/src/plugins/index.js

```diff
@@ -40,6 +40,7 @@ module.exports = {
   get 'jest-config' () { return require('../../../datadog-plugin-jest/src') },
   get 'jest-environment-node' () { return require('../../../datadog-plugin-jest/src') },
   get 'jest-environment-jsdom' () { return require('../../../datadog-plugin-jest/src') },
+  get '@jest/test-sequencer' () { return require('../../../datadog-plugin-jest/src') },
```

Comment:
Similar to the above, this should be in alphabetical order.

Reply:
fixed in https://github.com/DataDog/dd-trace-js/pull/3452/commits/02f357f6edc17df2f43d5fab10d02b0f7bc763c2